server/routes/tenants.js

```diff
@@ -33,4 +52,34 @@ router.post("/properties/:id/tenant", async(req, res) => {
     }
 })
 
-module.exports = router;
\ No newline at end of file
+// update Tenant
+router.put('/', async(req, res, next) => {
+    const { id, firstName, lastName, email, lease, propertyId } = req.body;
+    try {
+    const foundTenant = await Tenant.updateOne({tenantid: id}, {$set: 
+        {firstName: firstName,
+        lastName: lastName,
+        email: email,
+        lease: lease,
+        propertyId: propertyId}});
+
+        res.status(200).json(foundTenant);
+    } catch(e) {
+        console.error("What's the problem?", e);
+        res.status(500).json({error: "Error updating tenant"})
+    }
+    });
+
+// remove tenant
+router.delete('/:tenantId', async(req, res, next) => {
+    const id = req.params.tenantId;
+    try {
+        const deleteTenant = await Tenant.deleteOne({tenantId: id})
```

Comment:
```suggestion
        const deleteTenant = await Tenant.deleteOne({_id: id})
```
(test this works using mongodb)